src/main/webapp/profile.html

```diff
@@ -161,6 +161,14 @@ <h1 class="text-center">Your Profile</h1>
                 placeholder="" value="" name="linkedin">
           </div>
         </div>
+        <div class="form-check">
+          <input disabled class="form-check-input check-editable" type="checkbox" 
+              id="ok-shadow-check">
+          <label class="form-check-label" for="ok-shadow-check">
+            I am OK if another person shadows my interviews.
```

Comment:
Could this be "Allow another person to shadow my interviews" instead?

Reply:
Good idea. Will change.

Reply:
Good idea, will change.